src/NSrtm.Core/Level.cs

```diff
@@ -0,0 +1,9 @@
+namespace NSrtm.Core
+{
+    public enum Level
+    {
+        EllipsoidWgs84,
+        GeoidEgm2008,
+        Terrain
```

Comment:
I am aware, but such naming makes it much easier to reason about code, and facts you mentioned are not obvious for all team members - so such little redundancy is not problem.

Please fill extra `Levels` when it is required (feel free to update this branch).